enabler/package.json

```diff
@@ -18,6 +19,9 @@
     "sass": "1.70.0",
     "typescript": "5.3.3",
     "vite": "5.1.3",
-    "vite-plugin-css-injected-by-js": "3.4.0"
+    "vite-plugin-css-injected-by-js": "3.4.0",
+    "jest": "29.7.0",
+    "ts-jest": "29.1.2",
+    "ts-node": "10.9.2"
```

Comment:
is ts node really needed?

Reply:
https://github.com/commercetools/connect-payment-integration-template/pull/42/commits/99f56ee27a3d3ade66382697439a84c886a5b3f3